app/server-utils/stampy.ts

```diff
@@ -2,7 +2,12 @@ import {withCache} from '~/server-utils/kv-cache'
 import MarkdownIt from 'markdown-it'
 import MarkdownItFootnote from 'markdown-it-footnote'
 
-export type QuestionState = '_' | '-' | 'r'
+// export type QuestionState = '_' | '-' | 'r'
```

Comment:
if we don't need to keep commented-out code for some special reason, please delete.

Reply:
whoops - good catch